include/boost/graph/maximum_adjacency_search.hpp

```diff
@@ -149,7 +149,7 @@ namespace boost {
       //weight_type w;
       while (!pq.empty()) { // while PQ \neq {} do
         const vertex_descriptor u = pq.top(); // u = extractmax(PQ)
-        get(keys, u);                        vis.start_vertex(u, g);
+        /* weight_type w = */ get(keys, u);      vis.start_vertex(u, g);
```

Comment:
Should this comment still be here like this?

Reply:
See my comment above.